orienteer-camel/src/main/java/org/orienteer/camel/behavior/OIntegrationConfigStopBehavior.java

```diff
@@ -0,0 +1,42 @@
+package org.orienteer.camel.behavior;
+
+import org.apache.camel.CamelContext;
+import org.apache.camel.ServiceStatus;
+import org.apache.wicket.Component;
+import org.apache.wicket.behavior.Behavior;
+import org.apache.wicket.model.IWrapModel;
+import org.orienteer.camel.component.OIntegrationConfig;
+
+import ru.ydn.wicket.wicketorientdb.model.ODocumentModel;
+
+/**
+ * Behavior for {@link OIntegrationConfig.stop} OMethod
+ *
+ */
+public class OIntegrationConfigStopBehavior extends Behavior{
+
+	private static final long serialVersionUID = 1L;
+	
+	@Override
+	public void onConfigure(Component component) {
+		super.onConfigure(component);
+			ODocumentModel docModel=null;
+			if(component.getDefaultModel() instanceof ODocumentModel){
+				docModel = (ODocumentModel) component.getDefaultModel();
+			}else if(component.getDefaultModel() instanceof IWrapModel){
+				docModel = (ODocumentModel) ((IWrapModel<?>) component.getDefaultModel()).getWrappedModel();
```

Comment:
Why this if is needed? Theoretically component.getDefaultModelObject() should be more than enough. Also on "bind" you can check that behavior is just added to proper component. If behavior is just for specific component and will not be reused: recommend to use anonymous class.